BT-Tracking/Services/Reports/ReportService.swift

```diff
@@ -38,131 +36,193 @@ protocol ReportServicing: class {
 
 class ReportService: ReportServicing {
 
-    private var folderPattern: String = "exposure"
-    private var filePattern: String = "exposure.json"
+    var healthAuthority = "cz.covid19cz.erouska.dev"
```

Comment:
Nestačí, aby to byla interní konstanta? Proč by se měla měnit?

Reply:
Na produ bude jina

Reply:
Tak na to se víc hodí `#if DEBUG`

Reply:
Jo, ale prijde mi cistejsi to dat mimo ten kod

Reply:
Tak pak to dej to toho configu ve kterem udelas tento check na prostredi, z ktereho to budes brat jako konstantu. Nema smysl to mit jako var a jeste to mit v protokolu kdyz se to menit ani nema.